### PR TITLE
update way we check the event target

### PR DESCRIPTION
**Pull request by grasspoint**
Previous method was throwing an error on each click for IE11 and below

### CODE REVIEW

File: rocketbelt/components/buttons/rocketbelt.buttons-dropdown.js [hunk 1]
```diff
@@ -51,23 +51,10 @@
...
-        const el = path[j];
-
-        if (el.classList && (el.classList.contains('button-dropdown_content') || el.classList.contains('button-dropdown_icon') || el.classList.contains('button-dropdown_trigger'))) {
-          shouldClose = false;
-          break;
-        }
-      }
-
-      if (shouldClose) {
+      const el = $(e.target);
```
Comment (kendrick) [inline rocketbelt/components/buttons/rocketbelt.buttons-dropdown.js]: Can you factor `$` out of this? Trying to minimize jQ dependencies and we can pull off the next line without it.

File: rocketbelt/components/buttons/rocketbelt.buttons-dropdown.js [hunk 2]
```diff
@@ -51,23 +51,10 @@
...
+      if (!el.closest('.button-dropdown').length && !el.is('.button-dropdown')) {
```
Comment (kendrick) [inline rocketbelt/components/buttons/rocketbelt.buttons-dropdown.js]: "Closest" is built into modern browsers and RB polyfills it for older browsers, so we should only need to refactor `is` into `hasClass`.

Comment (grasspoint) [reply rocketbelt/components/buttons/rocketbelt.buttons-dropdown.js]: but hasClass won't work if el is not a jQuery object, right? or does RB already have a polyfill for that?

Comment (kendrick) [reply rocketbelt/components/buttons/rocketbelt.buttons-dropdown.js]: Whoops, got mixed up… meant `classList.contains`. Though you might need to use `classList.indexOf` for IE9 support… double check me. 😛

Comment (kendrick) [reply rocketbelt/components/buttons/rocketbelt.buttons-dropdown.js]: Yes, there's a polyfill for `closest` in the next RB release (which is ready any second now as soon as I write up the changelog). Sub in the [`development` version of `rocketbelt/base/rocketbelt.js`](https://github.com/Pier1/rocketbelt/blob/2516f691c6f6a35cfad632e4e9f3cd73cc6eeeb7/rocketbelt/base/rocketbelt.js#L94) if you'd like to test it out.

**Comment by kendrick** [comment]
Diff got me like

"wat"

**Pull request merged**